_posts/2019-12-09-ebcdic-character-format-guide.markdown

```diff
@@ -97,28 +80,40 @@ Let's start off showing a fictional fixed width copybook.
           10  FILLER               PIC X(50).
 ```
 
-WHAT DOES PIC MEAN?
+The spaces are important as that is a different section that is reserved for things such as comments.
+
+Next thing are the numbers these define the hierarchy of the copybook. In this example you have the top-level record and then all the fields contained with in it. Other uses could be and Address field that splits the Line 1, Line 2, and Country. Into sub sections. The numbers themselves don't actually matter, just needs to be larger than the parent.
+
+The next line is the name of the field.
 
-WHAT DOES BINARY DO?
+### PIC
 
-FILLER?
+PIC is the next thing we come across, this just tells the copybook how to interpret the data in this case using the value defined by the following number. Where `XXXX` or `X(4)` would be four characters.
 
-Copybooks are used to define the schema for an EBCDIC format. It defines the length and the format of the field.
+The basic types are the following:
 
 There are two main types:
 * X : Any character
 * 9 : An integer
 
+These fields then are specific modifiers for specific behaviour.
+
 There are also some special type information:
 * S : The field is packed
```

Comment:
It's easier to think of S as the sign rather than just indicating that the field is packed